File: Net/01_oop/02_interface/IUserPassword.cs
namespace OOP.Interface
{
    public interface IUserPassword
    {
        /// <summary>
        /// Implements <see cref="IUserPassword.UserPasswordChange(int, string, string)"/>
        /// </summary>
        /// <returns></returns>
        bool UserPasswordChange(int uid, string oldPassword, string newPassword);
    }
}

File: Java/src/p_1_oop/p_2_interface/IUserPassword.java
package p_1_oop.p_2_interface;

public interface IUserPassword {
    /**
     * checks if old password and new password are same
     * @param uid
     * @param oldPassword
     * @param newPassword
     * @return boolean
     */
    boolean userPasswordChange( int uid, String oldPassword, String newPassword);

}
